q2cli/tools.py

```diff
@@ -55,6 +70,9 @@ def export_data(path, output_dir):
               help='The format of the data to be imported. If not provided, '
                    'data must be in the format expected by the semantic type '
                    'provided via --type.')
+@click.option('--show-importable-types', is_flag=True, is_eager=True,
```

Comment:
> $ python -m unittest q2cli.tests.test_cli.CliTests produces failures for every test.

Use this command to run the unit tests:

```shell
QIIMETEST= nosetests
```

Take a look at `.travis.yml` to see how tests, linting, etc. are run by Travis-CI; you can use those same commands to test locally.

> You can see in my last commit that I started writing a test. Am I on the right track here? Is that something like what you're looking for?

You're on the right track! In addition to checking the exit code, we'd want to assert that some of the expected importable types are in the list (see some of the existing unit tests for how to assert that certain items are in the output).

The q2cli unit tests are written such that they will pass regardless of what plugins you may or may not have installed (e.g. q2-types, q2-feature-table, etc.). When you set the `QIIMETEST` environment variable (see the nosetests command above), that activates a "dummy" plugin defined in the QIIME 2 framework, and avoids loading any other plugins. The dummy plugin defines some importable types (in addition to other things), so you can assert that some of those importable types are in the CLI output.